integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -434,6 +436,9 @@ def move_page(request, region_slug, language_slug, page_id, target_id, position)
         # (The move()-method executes raw sql which might cause problems if the instance isn't fetched again)
         page = Page.objects.get(id=page_id)
         page.save()
+        # Moving tree nodes potentially causes changes to the fields tree_id, lft and rgt in the tree node model
+        # so the cache of page translations has to be cleared, because of it's relation to the page model
+        invalidate_model(PageTranslation)
```

Comment:
Sorry, another request: Could you mode this to the actual `move()`-method of the page model (you probably need to create this first and do a call to `super().move()` inside the new method)? Since moving can also be performed via the page form (not only drag & drop), I feel like that's the more flexible solution.

Reply:
Ah, yes, I missed the page form :see_no_evil: 
Thanks again for catching this